roles/internal/grok/tasks/install.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Clone Grok
+  git:
+    repo: https://github.com/GrokImageCompression/grok.git
+    dest: "{{ grok_clone_directory }}"
+    version: "{{ grok_version_tag }}"
+
+- name: Make build directory
+  file:
+    path: "{{ grok_clone_directory }}/build"
+    state: directory
+    mode: 0755
+
+- name: Cmake grok
+  cmake:
+    binary_dir: "{{ grok_clone_directory }}/build"
+    source_dir: "{{ grok_clone_directory }}"
+    build_type: Release
+      
+- name: Make grok
+  make:
+    chdir: "{{ grok_clone_directory }}/build"
+
+- name: Install grok
+  make:
+    chdir: "{{ grok_clone_directory }}/build"
+    target: install
+
+- name: Add Grok libraries to ld.so.conf.d
+  copy: src={{ item }} dest=/{{ item }} owner=root group=root mode=0444
+  with_items:
```

Comment:
See comment about this not being necessary above. If we do include it, I wouldn't use a loop to install one item. 

I'd rather see something like: 
```yaml
- name: Add Grok libraries to ld.so.conf.d
  copy: 
    src: libgrok.conf 
    dest: /etc/ld.so.conf.d/libgrok.conf 
    owner: root 
    group: root 
    mode: "0444"
```

or alternatively in the single line syntax
```yaml
- name: Add Grok libraries to ld.so.conf.d
  copy: src=libgrok.conf dest=/etc/ld.so.conf.d/libgrok.conf owner=root group=root mode=0444
```

I personally prefer the YAML syntax to the single line, but thats just a style thing.